nrf_security/cmake/kconfig_mbedtls_configure.cmake

```diff
@@ -50,6 +50,19 @@ macro(mbedtls_config_define_depends mbedtls_config)
   endforeach()
 endmacro()
 
+#
+# Internal macro which will enable the mbedtls configuration specified by the
+# first argument if at least one dependency is enabled
+#
+macro(mbedtls_config_define_depends_any mbedtls_config)
```

Comment:
This is not being used anywhere.

Please wait on introducing such macros until there is a need.

Reply:
Ok then, I will remote that when the CI is finished